buildSrc/src/main/groovy/nva.publication.channels.java.rootplugin.gradle

```diff
@@ -1,10 +1,27 @@
-plugins{
-    id 'nva.publication.channels.java.java-conventions'
+plugins {
+    id 'jacoco-report-aggregation'
+    id 'base'
+    id 'java-library'
     id 'nva.publication.channels.java.gradlelint'
-    id 'nva.publication.channels.java.jacoco-merge'
 }
 
 allprojects {
     //workaround for jacoco-merge to work
     apply plugin: 'nva.publication.channels.java.java-conventions'
 }
+
+reporting {
+    reports {
+        testCodeCoverageReport(JacocoCoverageReport) {
+            testType = TestSuiteType.UNIT_TEST
+        }
+        integrationTestCodeCoverageReport(JacocoCoverageReport) {
+            testType = TestSuiteType.INTEGRATION_TEST
+        }
```

Comment:
I wonder if this has an effect?

Reply:
could maybe combine if with a test report step in GitHub Actions for nice smooth reports? https://github.com/marketplace/actions/test-reporter